README.md

```diff
@@ -1,9 +1,9 @@
-# pints-models
+# bayesian-differential-equations-db
 
-The purpose of pints-models is to present Python implementations of scientific time series models from various disciplines.
+The purpose of bayesian-differential-equations-db is to present Python implementations of scientific time series models from various disciplines.
```

Comment:
I'd say something like, "The purpose of bayesian-differential-equations-db is to provide a database of differential equation models and associated observed time series data that represent interesting and challenging problems for inference. The database provides methods for running these models and accessing the associated observed data files using Python."